client/src/utils/optionalQueryString.ts

```diff
@@ -1,3 +1,7 @@
-export function optionalQueryString(value: string | undefined) {
-  return value ? String(value) : undefined;
+export function optionalQueryString(value: string | string[] | undefined) {
+  if (Array.isArray(value)) {
+    return String(value[0]).trim();
```

Comment:
it might be confusing - if you pass an array for serialisation you would expect to serialise it, not taking only first element. same for the fn in file below

Reply:
we can’t return the array because it’s expected to be string and we get errors
we return the first element of the array since ant design filters gives us the data as an array but with only one element

Reply:
right, I don't propose to return an array, but rather `serialise` it. ie to map array to string. I understand that antd might return array, but this function doesn't have anything related with antd(it is located in `utils` and makes `query string`. That's why it looks confusing to me.

Reply:
do I get you correctly that you propose to iterate through the array and make it as a string. Smth like this ```value.reduce((acc, cur) =>  `${acc} ${cur}`,'').trim().```

Reply:
or do you mean just do a regular join  `array.join('')` ?

Reply:
I guess the better option would be array.join(',')